### ISSUE TITLE
Add static language resources

### ISSUE THREAD

**Issue by gitgash** [bmabey/pyLDAvis#22]
Add static language resources to display static titles in different languages.
Add current language detection for session and choose appropriate language bundle to display

**Comment by bmabey** [comment]
I agree that adding internationalization would be helpful. I won't be doing this but will accept a PR.  

Language detection doesn't seem like it would be worth the effort and complexity added to the codebase.
